R/lintr-package.R

```diff
@@ -8,11 +8,11 @@
 "_PACKAGE"
 
 ## lintr namespace: start
-#' @importFrom glue glue
+#' @importFrom glue glue glue_collapse glue_data
 #' @importFrom rex rex regex re_matches re_substitutes character_class
 #' @importFrom stats na.omit
 #' @importFrom utils capture.output head getParseData relist
-#' @importFrom xml2 xml_find_all as_list
+#' @importFrom xml2 xml_attr xml_attrs xml_find_all xml_find_chr xml_find_num xml_find_first xml_text as_list
```

Comment:
`xml_attrs()` is only used twice in a single line. Not worth importing IMO.
Similiarly, `glue_data()` only has one usage.

Reply:
I think there's no cost, but might keep on namespace-qualifying.

OTOH, I think the main advantage of doing so is readability, I'm not sure any contributor can get too tripped up guessing where `xml*` and `glue*` functions come from internally (`xmlparsedata::xml_parse_data()` is a good counterexample where we should definitely qualify).

Anyway I don't feel strongly; added back the qualification.

Reply:
Can you remove the import as well?

Reply:
I would leave it in. I think it's better for dependencies to be incurred at build time than run time, where appropriate. I don't think the current NAMESPACE is exhaustive in that regard, but this is a good start.

Reply:
Then I'd rather not ns-qualify their usages, it seems counter-intuitive to have an unused explicit import.

Reply:
OK... I get that it feels redundant. In this case, I think of `::` as being an emphasis for readability. "Yes, we imported it, but just in case you forgot, here's where it came from". Seems appropriate for "uncommon" or infrequent calls.

Anyway, I'll just remove the imports, since I do think the readability aspect is more important in this case.